src/scrabble.js

```diff
@@ -1,5 +1,44 @@
-function scrabble() {
+const letternum = {
```

Comment:
It might be a good idea to declare the letternum variable outside of the scrabble() scope